Client/mods/deathmatch/logic/CClientPad.h

```diff
@@ -35,18 +35,18 @@ class CClientPad
     void DoPulse(CClientPed* pPed);
 
     static bool GetAnalogControlState(const char* szName, CControllerState& cs, bool bOnFoot, float& fState, bool bIgnoreOverrides);
-    static bool SetAnalogControlState(const char* szName, float fState);
+    static bool SetAnalogControlState(const char* szName, float fState, bool bFrameForced);
     static void RemoveSetAnalogControlState(const char* szName);
 
     static void ProcessSetAnalogControlState(CControllerState& cs, bool bOnFoot);
-    static void ProcessControl(short& usControlValue, unsigned int uiIndex, bool bResetCmp);
+    static void ProcessControl(short& usControlValue, unsigned int uiIndex); //, bool bPositive
```

Comment:
I'm unsure what this comment is supposed to mean?

Reply:
overlooked it while cleaning up :x
i also made some small refactors while working on the stuff and removed some redundant code